src/app/app.module.ts

```diff
@@ -21,8 +22,9 @@ import { DataStoreService } from './services/data-store.service';
     HomeComponent,
     PageNotFoundComponent,
     HeaderComponent,
-    SearchUserComponent,
-    ShowUserComponent
+    UserSearchComponent,
+    ShowUserComponent,
```

Comment:
This should not be named `ShowUserComponent` since its going to have the update functionality in future as well. It should be named `UserListComponent`